testnets/neutrontestnet/chain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "neutrontestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Neutron Testnet",
+  "chain_id": "baryon-1",
+  "bech32_prefix": "neutron",
+  "daemon_name": "neutrond",
+  "node_home": "$HOME/.neutrond",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "untrn",
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.05
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "untrn"
```

Comment:
neutron isn't a staking token innit?

Reply:
I thought that it might be used for compatibility with different wallets etc. But agree, we can remove it.

Reply:
I think we'd better remove it to avoid misunderstanding. Let's return it once (if) we'll face any related issues